### ISSUE TITLE
FR: Range slider

### ISSUE THREAD

**Issue by jalal007** [ampproject/amphtml#17831]
## What's the issue?

Hi, I'm developing a mobile site with AMP project.  My project require a price range slider based data searching like the airbnb have, to see the airbnb Price Range slider, please [Click here](https://github.com/airbnb/rheostat) 

I didn't find any Price range slider on AMP yet which will fulfill my need. 

Could you please check the link i have attached and let me know your concern including how can i implement a price range slider on my project with AMP as like airbnb have.

**Comment by aghassemi** [comment]
I am thinking this should be possible without a need for a dedicated component. Overlaying two `<input  type=range>` elements and using `amp-bind` to change the `min` and `max` values of each  as their values changes should accomplish this ( `min` and `max` may not be bindable at the moment but that's a super easy fix ).

**Comment by ericlindley-g** [comment]
Excellent!  /cc @kul3r4 @sebastianbenz @fstanis for visibility w/r/t AMP by Example

**Comment by sebastianbenz** [comment]
@aghassemi I played around with this, but couldn't get it to work. The problem is that only one slider will receive touch events: https://codepen.io/anon/pen/PdKmZe

**Comment by aghassemi** [comment]
@sebastianbenz Found an example that uses pointer-events to fix that issue: see https://codepen.io/sakamies/pen/qOEGLJ 
essentially it disable pointer-events on the input and re-enables it on the thumb only. Looks promising.

**Comment by sebastianbenz** [comment]
Ouh - that's clever! Wasn't even aware that you could target the thumb
specifically. Works: https://codepen.io/anon/pen/NLvwyr

...this will make a nice sample!

**Comment by caroqliu** [comment]
Closing due to workaround described above
